packages/grid/x-data-grid-generator/src/useMovieData.ts

```diff
@@ -0,0 +1,284 @@
+    composer: {
+      name: 'Brian Tyler',
+    },
+  },
+  {
+    title: 'Frozen II',
+    gross: 1450026933,
+    director: 'Chris Buck & Jennifer Lee',
+    company: 'Disney Studios',
+    year: 2019,
+    cinematicUniverse: 'Frozen',
+    composer: {
+      name: 'Christophe Beck',
+    },
+  },
+  {
+    title: 'Avengers: Age of Ultron',
+    gross: 1402804868,
+    director: 'Joss Whedon',
+    company: 'Disney Studios',
+    year: 2015,
+    cinematicUniverse: 'Marvel Cinematic Universe',
+    composer: {
+      name: 'Danny Elfman',
+    },
+  },
+  {
+    title: 'Black Panther',
+    gross: 1347280838,
+    director: 'Ryan Coogler',
+    company: 'Disney Studios',
+    year: 2018,
+    cinematicUniverse: 'Marvel Cinematic Universe',
+    composer: {
+      name: 'Ludwig Göransson',
+    },
+  },
+  {
+    title: 'Harry Potter and the Deathly Hallows – Part 2',
+    gross: 1342025430,
+    director: 'David Yates',
+    company: 'Warner Bros. Pictures',
+    year: 2011,
+    composer: {
+      name: 'Alexandre Desplat',
+    },
+  },
+  {
+    title: 'Star Wars: The Last Jedi',
+    gross: 1332539889,
+    director: 'Rian Johnson',
+    company: 'Disney Studios',
+    year: 2017,
+    cinematicUniverse: 'Star Wars',
+    composer: {
+      name: 'John Williams',
+    },
+  },
+  {
+    title: 'Jurassic World: Fallen Kingdom',
+    gross: 1309484461,
+    director: 'J. A. Bayona',
+    company: 'Universal Pictures',
+    year: 2018,
+    cinematicUniverse: 'Jurassic Park',
+    composer: {
+      name: 'Michael Giacchino',
+    },
+  },
+  {
+    title: 'Frozen',
+    gross: 1290000000,
+    director: 'Chris Buck & Jennifer Lee',
+    company: 'Disney Studios',
+    year: 2013,
+    cinematicUniverse: 'Frozen',
+    composer: {
+      name: 'Christophe Beck',
+    },
+  },
+  {
+    title: 'Beauty and the Beast',
+    gross: 1263521136,
+    director: 'Bill Condon',
+    company: 'Disney Studios',
+    year: 2017,
+    composer: {
+      name: 'Alan Menken',
+    },
+  },
+  {
+    title: 'Incredibles 2',
+    gross: 1242805359,
+    director: 'Brad Bird',
+    company: 'Disney Studios',
+    year: 2018,
+    composer: {
+      name: 'Michael Giacchino',
+    },
+  },
+  {
+    title: 'The Fate of the Furious',
+    gross: 1238764765,
+    director: 'F. Gary Gray',
+    company: 'Universal Pictures',
+    year: 2017,
+    cinematicUniverse: 'Fast & Furious',
+    composer: {
+      name: 'Brian Tyler',
+    },
+  },
+  {
+    title: 'Iron Man 3',
+    gross: 1214811252,
+    director: 'Shane Black',
+    company: 'Disney Studios',
+    year: 2013,
+    cinematicUniverse: 'Marvel Cinematic Universe',
+    composer: {
+      name: 'Brian Tyler',
+    },
+  },
+  {
+    title: 'Minions',
+    gross: 11159398397,
+    director: 'Pierre Coffin & Kyle Balda',
+    company: 'Universal Pictures',
+    year: 2015,
+    composer: {
+      name: 'Heitor Pereira',
+    },
+  },
+];
+
+const getRowId = (row: any) => row.title;
```

Comment:
Using the `title` as the identifier may not be a good idea. If I set `editable=true` in the Title column, every time I change a movie title it gets duplicated. Adding a hidden id column is better.

Reply:
I did not think about the editing while building it but you are right

Reply:
Done :+1: